circuits.go

```diff
@@ -34,6 +35,9 @@ const (
 // ErrorCircuitIDNotFound returns if CircuitID is not registered
 var ErrorCircuitIDNotFound = errors.New("circuit id not supported")
 
+var errUserStateIsNotValid = errors.New("user state is not valid")
```

Comment:
Maybe, errors should be public? In case of error handling on the client-side, it would be convenient.